README.md

```diff
@@ -23,7 +25,7 @@ The bot can be configured only with environment variables. For now there are we
 | THRESHOLD | Sets a threshold for code detection algorithm | Any valid `u8` | `3` | All mods |
 | BIND_ADDRESS | Address for binding the web-service | Any valid IP address | `0.0.0.0` | Webhook mode |  
 | BIND_PORT | Port for binding the web-service | Any valid port | `8080` | Webhook mode |
-| HOST | Host, where Telegram will send updates in webhook mode | Any valid host address | None | Webhook mode |
+| WEBHOOK_URI | This variable allows you to set your hook path example **https://example.org/api/v1/Te3_@#ge** |Any valid host+path | None | Webhook mode |
 
 If for any variable there is no default value and you didn't provide any value - the bot won't start.
 Bot automatically registers webhook (if is launched in webhook mode) with address `https://$HOST/$TELOXIDE_TOKEN/api/v1/message`.
```

Comment:
For now this line isn't true. So you have two options:
1. Remove this line since it will b a lie
2. Add any default route to the bot, if URI wasn't provided. Default route must be generated in some anti-bruteforce manner (random UUID, which is URI-valid, is totally fine). Don't forget to log it, since this URI can be required for load-balancer configuration.

Really I prefer the second approach. it's quite a lot work and can be done in another PR.